Make.inc

```diff
@@ -1083,24 +1083,24 @@ endif
 endif
 
 ifeq ($(USE_SYSTEM_LIBM), 1)
-LIBM := -lm
-LIBMNAME := libm
+LIBM ?= -lm
```

Comment:
Maybe this is a bit too much, but custom libm's do exist...

Reply:
We don't use the libm for much - so it doesn't really matter, and often llvm dispatches to the system libm anyways. For example see https://github.com/JuliaLang/julia/pull/42299